wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,202 @@
-
-**Save and Reset**
-
--   **Ctrl+L**: Revert to saved
--   **Ctrl+S**: Quick save
--   **Ctrl+N**: Remove all notes (but keep timing/song setup)
-
-**Mode switch**
-
--   **F1**: Compose mode
--   **F2**: Design mode (only basics available at the moment)
--   **F3**: Timing mode
--   **F4**: Song setup
+# Shortcut Key Reference
+
+There are many shortcuts in osu! that can be utilized while in-game or while idle at any screen.
+They are listed out with their functions below.
+
+**Note:** some of these keys can be changed in the [Options](/wiki/Options/)!
+However, the list below presents the default keys.
+
+## Game-Wide
+
+These shortcuts work anywhere:
+
+- `Arrow Down`: Decrease volume.
+- `Arrow Up`: Increase volume.
+- `Alt` + `F4`: If playing/editing a beatmap, exit to song select; otherwise, this will close the osu!client.
+- `Ctrl` + `Alt` + `Shift` + `S`: Reload current skin (may say that it is in queue)
+- `Ctrl` + `F11`: Toggle trame times
+- `Ctrl` + `O`: Open options menu (does not work while playing in-game)
+- `F7`: Toggle FPS limiting mode
+- `F8`: Toggle chat window
+- `F9`: Toggle extended chat window
+- `F10`: Toggle all mouse buttons (except for mouse wheel)
+- `F12`: Take a screenshot (hold `Shift` to upload instantly to the server for sharing.)
+- `Insert`: Minimize the osu!client into the notification area.
+- `Shift` + `F1`: Open [osu!status](https://twitter.com/osustatus "osu!status on twitter") (when connection error appears)
+
+## Main Menu
+
+- `Alt` + `Enter`: Toggle fullscreen mode
+- `Esc`: Go back one menu
+- `P`: Play mode
+- `P`, then `P`: Solo
+- `P`, then `M`: [Multi](/wiki/Multi)
+- `E`: Edit mode
+- `Z` or `F1`: Previous song (jukebox)
+- `X` or `F2`: Play from start (jukebox)
+- `C`: Toggle pause (jukebox)
+- `F4`: Stop playing and reset duration to 0 (jukebox)
+- `V` or `F5`: Next song (jukebox)
+- `J` or `F6`: Jump To (jukebox)
+- `R`: Random song (jukebox)
+- `D`: osu!direct ([osu!supporters](/wiki/osu!supporter) only)
+- `O`: Options
+- `Q` or `Esc`, then `Esc`: Quit (displays a prompt)
+
+## Song Select
+
+- `F1`: Display [mod](/wiki/Game_Modifiers) selection screen
+- `F2`: Random map selection
+- `Shift` + `F2`: Undo random map selection
+- `F3`: Display beatmap options
+- `F5`: Refresh listing (repair mode)
+- `Shift` + `Delete`: Delete current map
+- `Ctrl` + `1`: Switch to [osu!](/wiki/osu!)
+- `Ctrl` + `2`: Switch to [osu!taiko](/wiki/osu!taiko)
+- `Ctrl` + `3`: Switch to [osu!catch](/wiki/osu!catch)
+- `Ctrl` + `4`: Switch to [osu!mania](/wiki/osu!mania)
+- `Ctrl` + `Enter`: Play current map with [Auto](/wiki/Auto) mod
+- `Ctrl` + `Shift` + `Enter`: Play current map with [Cinema](/wiki/Cinema) mod
+- `Ctrl` + `A`: Toggle Auto mod
+- `Ctrl` + `F3` or `Ctrl` + `+`: Increase osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `F4` or `Ctrl` + `-`: Decrease osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `Backspace`: Delete last word (when using search)
```

Comment:
ctrl is a modifier for selecting and deleting whole words in windows, why mention?